packages/xml-views-quick-fix/test/quick-fix-id-spec.ts

```diff
@@ -0,0 +1,171 @@
+      const quickFixInfo = getQuickFixIdInfo(testXmlDoc, { start, end });
+      expectExists(quickFixInfo, "Quick Fix Info");
+      expect(quickFixInfo.suggestion).to.equal(expectedSuggestion);
+      expect(quickFixInfo.offsetRange.start).to.equal(idStartOffest);
+    });
+
+    it("will get quick fix info when class has empty id attribute", () => {
+      const testXmlSnippet = `<mvc:View
+                                xmlns:mvc="sap.ui.core.mvc"
+                                xmlns="sap.m"> 
+                                <mvc:content>
+                                  <🢂List🢀 $id=""></List>
+                                </mvc:content>
+                              </mvc:View>`;
+      const expectedSuggestion = 'id="_IDGenList1"';
+      const { document, start, end, idStartOffest } = getXmlSnippet(
+        testXmlSnippet
+      );
+      const testXmlDoc = getXmlDocument(document);
+      const quickFixInfo = getQuickFixIdInfo(testXmlDoc, { start, end });
+      expectExists(quickFixInfo, "Quick Fix Info");
+      expect(quickFixInfo.suggestion).to.equal(expectedSuggestion);
+      expect(quickFixInfo.offsetRange.start).to.equal(idStartOffest);
+    });
+
+    it("will get quick fix info when class has empty id attribute between other attributes", () => {
+      const testXmlSnippet = `<mvc:View
+                                xmlns:mvc="sap.ui.core.mvc"
+                                xmlns="sap.m"> 
+                                <mvc:content>
+                                  <🢂List🢀 models="" $id="" footerText=""></List>
+                                </mvc:content>
+                              </mvc:View>`;
+      const expectedSuggestion = 'id="_IDGenList1"';
+      const { document, start, end, idStartOffest } = getXmlSnippet(
+        testXmlSnippet
+      );
+      const testXmlDoc = getXmlDocument(document);
+      const quickFixInfo = getQuickFixIdInfo(testXmlDoc, { start, end });
+      expectExists(quickFixInfo, "Quick Fix Info");
+      expect(quickFixInfo.suggestion).to.equal(expectedSuggestion);
+      expect(quickFixInfo.offsetRange.start).to.equal(idStartOffest);
+    });
+  });
+
+  context("true negative scenarios", () => {
+    it("will not suggest quick fix id when the tag is empty", () => {
+      const testXmlSnippet = `<mvc:View
+                                xmlns:mvc="sap.ui.core.mvc"
+                                xmlns="sap.m"> 
+                                  <mvc:content>
+                                    <🢂🢀></List>
+                                  </mvc:content>
+                              </mvc:View>`;
+      const { document, start, end } = getXmlSnippet(testXmlSnippet);
+      const testXmlDoc = getXmlDocument(document);
+      const quickFixInfo = getQuickFixIdInfo(testXmlDoc, {
+        start,
+        end,
+      });
+
+      expect(quickFixInfo).to.not.exist;
+    });
+
+    it("will not suggest quick fix id when the tag is empty", () => {
```

Comment:
the desciption string `will not suggest quick fix id when the tag is empty` appears twice in this file